src/ucs/type/status.h

```diff
@@ -74,6 +74,7 @@ typedef enum {
     UCS_ERR_UNSUPPORTED            = -22,
     UCS_ERR_REJECTED               = -23,
     UCS_ERR_NOT_CONNECTED          = -24,
+    UCS_ERR_CONNECTION_RESET       = -25,
```

Comment:
this status is also added in PR - https://github.com/openucx/ucx/pull/4702

Reply:
yes, added here as well to avoid compilation errors. This copy-paste-code should be successfully merged by GIT or easy to resolve when #4702 is merged.